telegram/files/sticker.py

```diff
@@ -72,5 +83,67 @@ def de_json(data, bot):
         data = super(Sticker, Sticker).de_json(data, bot)
 
         data['thumb'] = PhotoSize.de_json(data.get('thumb'), bot)
+        data['mask_position'] = MaskPosition.de_json(data.get('mask_position'), bot)
 
         return Sticker(**data)
+
+    @staticmethod
+    def de_list(data, bot):
+        if not data:
+            return list()
+
+        stickers = list()
```

Comment:
It's better to use comprehensions etc here like:
stickers = [Sticker.de_json(ix, bot) for ix in data]